website/src/css/custom.css

```diff
@@ -195,11 +422,32 @@ nav.navbar div.navbar__inner {
   font-size: 20px !important;
 }
 
-/* Shamelessly copy/pasted from our style guide */
+/* admonition */
+.admonition {
+  border-left-width: 0.625rem;
+  border-radius: 0.625rem;
+}
```

Comment:
The "note" admonition styles do not update in dark mode. Example page: https://deploy-preview-1990--docs-getdbt-com.netlify.app/blog/change-data-capture

![image](https://user-images.githubusercontent.com/13647110/190257681-5c2d735c-43ec-4448-93e1-9f933e40f278.png)

Reply:
should be working now:
https://deploy-preview-1990--docs-getdbt-com.netlify.app/blog/change-data-capture